omnipaxos/src/ballot_leader_election.rs

```diff
@@ -225,15 +229,17 @@ impl BallotLeaderElection {
     }
 
     pub(crate) fn hb_timeout(&mut self) -> Option<Ballot> {
-        let my_connectivity = self.ballots.len() + 1;
+        let my_connectivity = self.ballots_temp.len() + 1;
         self.connectivity = my_connectivity as Connectivity;
+        self.ballots_temp
+            .push((self.current_ballot, self.connectivity));
+        self.ballots = std::mem::take(&mut self.ballots_temp);
```

Comment:
Similarly here: I don't think we should change the core BLE code. Instead, add what you need under the `ui` feature